File: 프로그래머스/1/42748. K번째수/K번째수.js
function solution(array, commands) {
    let answer = [];
    
    for (let i = 0; i < commands.length; i++) {
        const sliceFrom = commands[i][0];
        const sliceTo = commands[i][1];
        const sortedArrayIndex = commands[i][2];
        
        const slicedArray = array.slice(sliceFrom - 1, sliceTo);
        const sortedArray = slicedArray.sort((a,b) => a - b);
        answer.push(sortedArray[sortedArrayIndex - 1]);
    }
    
    return answer;
}

File: 프로그래머스/1/42748. K번째수/K번째수.py
def solution(array, commands):
    
    answer = []
    
    for command in commands:
        start = command[0]
        end = command[1]
        order = command[2]
        
        slicedArray = array[start-1:end]
        slicedArray.sort()
        answer.append(slicedArray[order-1])
    
    return answer
        
        
        